app/Controllers/SettingsController.php

```
@@ -570,6 +579,20 @@ private function templateDefaults(): array
         return $defaults;
     }
 
+    private function resolveTemplateEditorLocale(mixed $requested): string
+    {
+        $available = \App\Support\I18n::getAvailableLocales();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\I18n' in method 'resolveTemplateEditorLocale'. (undefined)

(StaticAccess)